docs/content/manual/pre-release/node/backing-image-on-a-down-node.md

```diff
@@ -3,18 +3,18 @@ title: Backing Image on a down node
 ---
 
 1. Update the settings:
-   1. Disable Node Soft Anti-affinity.
-   2. Set Replica Replenishment Wait Interval to a relatively long value.
-2. Create a backing image:
+   1. Disable `Node Soft Anti-affinity`.
+   2. Set `Replica Replenishment Wait Interval` to a relatively long value.
+2. Create a backing image. Wait for the backing image being ready in the 1st disk.
```

Comment:
Should we expect the backing image pod to be created at this moment?

Reply:
At this point, if the node containing the backing image is powered down, will the system create another image on another node?
My understanding is if the volume is not attached to backing image, it will maintain at least 1 image on running disk.

Reply:
1. A backing image file will be downloaded then stored in a backing-image-manager pod immediately once a backing image is created, no matter if there is a volume using it.
2. For downloaded backing image, YES. Longhorn should re-download it if the node contains the only backing image file is down. But for uploaded backing image in the future, there is no way to do auto recovery